views/support/faqbrowse.tt

```diff
@@ -0,0 +1,109 @@
+<script language='JavaScript'>
+<!--
+    function showAnswer() {
+        if (!document.getElementById) {
+            return true;
+        }
+        var a = document.getElementById('answer');
+        var r = document.getElementById('rm');
+        if (!a || !r) {
+            return true;
+        }
+        r.style.display = 'none';
+        a.style.display = 'block';
+        return false;
+    }
+//-->
+
+
+</script>
+[%- CALL dw.active_resource_group("foundation") -%]
+[%- dw.need_res({ group => "foundation"}
+        "stc/faq.css"
+        ) -%]
+
+
+
+[%- UNLESS faqs -%]
+    [%- sections.title = title -%]
+    <p><b>[% '.error.nofaq' | ml %]</b></p>
+[%- ELSE -%]
+    [%- sections.title = title -%]
+<div id='faq'>
+    [%- IF faqcatarg -%]
+        <ol>
+            [%- FOR f IN faqs -%]
+                <li><a href=[%- "#faq${f.faqid}" -%]>[% f.question %]</a></li>
+            [%- END -%]
+        </ol>
+    [%- END -%]
+    [%- FOR f IN faqs -%]
+        [%- IF faqcatarg -%]
+            <h1>
+                <a id='faq[% f.faqid %]'></a>
+                <a style='text-decoration: none;' href='faqbrowse?faqid=[% f.faqid %]'>&#x00bb; [%- f.question -%]</a>
+            </h1>
+        [%- END -%]
+        <div style='margin-left: 20px;'>
+            [%- IF f.display_summary -%]
+                <div class='summary'>[% f.summary %]</div>
+                [% UNLESS f.display_answer %]
+
+                    <div class='rm'>
+                        [%- oc = faqcatarg ? '' : "onclick='return showAnswer();'" -%]
+                        <b>( <a [% f.oc %] href='faqbrowse?faqid=[% "$f.faqid$q" %]&view=full'>[% '.more' | ml %]</a> )</b>
```

Comment:
That `"$f.faqid$q"` interpolation-thing looks suspect; would probably be better as multiple prints?

Reply:
So, it works, which means it's a question of whether it'd be more readable as multiple prints. I tend to feel like having a bunch of separate TT prints next to each other feels clunky and doesn't give as clear an idea of what the final result is, but I'm open to other opinions.

Reply:
Ok cool! Now that my brain isn't attuned to hunting BMLisms, it kind of looks fine again, so never mind.